api/main.js

```diff
@@ -10,7 +10,7 @@ const routes = require('./routes');
 const server = express();
 
 server.use(express.urlencoded({ extended: true }));
-server.use(cors());
+server.use(cors({ credentials: true, origin: true }));
```

Comment:
:tada:

Reply:
Now that I've thought about this a little more, and after doing some reading about what CORS is even supposed to be, I think the right solution from a security standpoint is to set the origin to a static value.  It'd probably have to come from an environment variable so we could vary it based on what was consuming it.  CORS still isn't super helpful, but 🤷‍♂️ 

Anyway, I'll open a backlog issue for that, this is perfectly fine for now.